StripeFinancialConnections/StripeFinancialConnections/Source/Native/InstitutionPicker/InstitutionTableViewCell.swift

```diff
@@ -10,12 +10,13 @@ import Foundation
 import UIKit
 
 final class InstitutionTableViewCell: UITableViewCell {
+    private var theme: FinancialConnectionsTheme!
```

Comment:
RE: https://github.com/stripe/stripe-ios/pull/3811#discussion_r1686967617

oh maybe this is why you had the nullls? (note that I could have been wrong where NULL was justified 😅 )

hmm it's hard to have the full picture without playing around with it but wouldn't this crash if something is wrongly accessed? for example any access to `institutionCellView` can crash if not accessed after `customize`? 

I can approve if you are OK with the risks, but I do feel like something likely has to be re-organized because there are several oddities that usually don't have to happen:
1. use of `!` (I almost never use it due to the risks)
2. I believe the moving of `contentView.addAndPinSubview(institutionCellView)` likely _works_ (its a no-op) because its just "re-adding" the same view(?) but it might be "logically wrong" because you are only supposed to add the view once (ex. table view reloads, but it reuses the same cells)
3. lazy property that requires `theme` to be there, but the `theme` is not passed in initializer, its in `customize`

overall, the core problem seems to be that the introduction of `theme` is breaking how the code existed today, and this is causing ripple effects

for example, one mind shift could be that: maybe due to the introduction of `theme` and the need/want for the theme to be passed in initializers (and UITableView not allowing to pass theme in init?), we can no longer construct the view in `init`; instead we need to construct it in `customize`, and this means how we build `InstitutionTableViewCell.swift` has to change from first-principles (ex. lazy variables, and non-nulls of _views_, potentially don't make sense because we don't have the info to construct them until `customize` is called etc. etc.)

---

💡  thinking out loud to unblock: a quick patch could be to just assign a theme `private var theme: FinancialConnectionsTheme = .light` - its still odd but at the very least avoids a crash

Reply:
Yeah this one is a bit tricky, because we need the `theme` before we add the view, but we can't pass it in an initializer, since we're using an implicit `UITableViewCell` initializer: 

```swift
let cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier) as? InstitutionTableViewCell
```

As for the force unwrapped property, I agree with your suggestion of changing it from a `lazy` var, and just making it an optional var that is set in the `customize` call. This way we don't even need to store `theme` as a property on `self`!